docs/v3/core/v3.0.rst

```diff
@@ -1112,40 +1065,26 @@ A Zarr group metadata object must contain the following mandatory key:
 ``zarr_format``
 ^^^^^^^^^^^^^^^
 
-    Must be the string "https://purl.org/zarr/spec/v3#group".
+    An integer defining the version of the storage specification to which the
+    array store adheres, muste be ``3`` here.
 
 Optional keys:
 
-``extensions``
-^^^^^^^^^^^^^^
+``storage_transformers`` (group)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    TBD
 
-    An array containing zero or more objects, each of which identifies
-    an extension and provides any additional extension
-    configuration metadata. Each object must contain the name
-    ``extension`` whose value is a URI that identifies a Zarr
-    extension and dereferences to a human readable representation of
-    the extension specification. Each object must also contain the
-    name ``must_understand`` whose value is either the literal
-    ``true`` or ``false``. Each object may also contain the name
-    ``configuration`` whose value is defined by the
-    extension.
-
-    If an implementation of this specification encounters an extension
-    that it does not recognize, but the value of ``must_understand``
-    is ``false``, then the extension may be ignored and processing may
-    continue. If the extension is not recognized and the value of
-    ``must_understand`` is ``true`` then processing must terminate and
-    an appropriate error raised.
-
-``descenents_may_have_group_extensions``
+``descenents_must_be_read``
```

Comment:
Note: My original name had a typo, "descenents" instead of "descendents".

However, I would actually propose that in the interest of getting v3 out sooner, we leave out this property for now so that we can consider it more carefully after getting out v3 but before adding any group extensions.  If we add this property later, and an implementation is unaware of it, it will fail if it encounters it, which is correct behavior, since if it doesn't know about this property then it also won't know about the group extensions.   There are some edge cases where technically the implementation might be able to read parts of the group successfully even without knowing about this property, but I don't think we have to worry about that.

Reply:
> However, I would actually propose that in the interest of getting v3 out sooner, we leave out this property for now so that we can consider it more carefully after getting out v3 but before adding any group extensions. If we add this property later, and an implementation is unaware of it, it will fail if it encounters it, which is correct behavior, since if it doesn't know about this property then it also won't know about the group extensions. There are some edge cases where technically the implementation might be able to read parts of the group successfully even without knowing about this property, but I don't think we have to worry about that.

Good idea! I added a separate issue for it: #207